src/ExpressiveInstaller/OptionalPackages.php

```diff
@@ -472,8 +537,8 @@ public static function removeLinesContainingStrings(array $entries, $content)
     /**
      * Ask if the user would like a minimal install.
      *
+     * @deprecated since 1.1.0, and no longer used internally
```

Comment:
Why deprecated? For a normal project I understand why public methods are deprecated and later removed. However this installer self destructs itself after it's ready. So in theory this installer never can get extended and its public methods can never get called after installation.